source/_components/telegram_bot.markdown

```diff
@@ -79,3 +100,118 @@ action:
+    - service: notify.telegram
+      data_template:
+        title: '*Dumb automation*'
+        target: '{{ trigger.event.data.user_id }}'
+        message: 'You said: ``` {{ trigger.event.data.text }} ```'
+        data:
+          disable_notification: true
+          inline_keyboard:
+            - '/edit,/NO'
+            - '/remove button'
+```
+
+Message editor:
+```yaml
+- alias: 'telegram bot last sended msg edit'
+  hide_entity: true
+  trigger:
+    platform: event
+    event_type: telegram_callback
+    event_data:
+      data: '/edit'
+  action:
+    - service: notify.telegram
+      data_template:
+        target: '{{ trigger.event.data.user_id }}'
+        message: 'Editing the message!'
+        data:
+          callback_query:
+            callback_query_id: '{{ trigger.event.data.id }}'
+            show_alert: true
+    - service: notify.telegram
+      data_template:
+        title: '*Message edit*'
+        target: '{{ trigger.event.data.user_id }}'
+        message: >
+          Callback received from {{ trigger.event.data.from_first }}.
+          Message id: {{ trigger.event.data.message.message_id }}.
+          Data: ``` {{ trigger.event.data.data }} ```
+        data:
+          edit_message:
+            message_id: '{{ trigger.event.data.message.message_id }}'
+          disable_notification: true
+          inline_keyboard:
+            - '/edit,/NO'
+            - '/remove button'
+```
+
+Keyboard editor:
+```yaml
+- alias: 'telegram bot keyboard edit'
+  hide_entity: true
+  trigger:
+    platform: event
+    event_type: telegram_callback
+    event_data:
+      data: '/remove button'
+  action:
+    - service: notify.telegram
+      data_template:
+        target: '{{ trigger.event.data.user_id }}'
+        message: 'Callback received for editing the inline keyboard!'
+        data:
+          callback_query:
+            callback_query_id: '{{ trigger.event.data.id }}'
+            show_alert: false
+    - service: notify.telegram
+      data_template:
+        target: '{{ trigger.event.data.user_id }}'
+        message: ''  # this is needed for the general hass notify service
+        data:
+          edit_replymarkup:
+            message_id: 'last'
+          disable_notification: true
+          inline_keyboard:
+            - '/edit,/NO'
+
+```
+
+Only acknowledges the 'NO' answer:
+```yaml
+- alias: 'telegram bot simply acknowledges'
+  hide_entity: true
+  trigger:
+    platform: event
+    event_type: telegram_callback
+    event_data:
+      data: '/NO'
+  action:
+    - service: notify.telegram
+      data_template:
+        target: '{{ trigger.event.data.user_id }}'
+        message: 'OK, you said no!'
+        data:
+          callback_query:
+            callback_query_id: '{{ trigger.event.data.id }}'
+            show_alert: false
+```
+
+For a more complex usage of the `telegram_bot` capabilities, using [AppDaemon](https://home-assistant.io/docs/ecosystem/appdaemon/tutorial/) is advised.
```

Comment:
Thanks a lot for your review. I'm going to try again. 

On the AppDaemon tool, it is my intention to link or include in the documentation a simple example, perhaps with the same behavior of the yaml examples. My uses are too complex to use as examples...

Do you think the screenshots are necessary too? (https://github.com/home-assistant/home-assistant/pull/7294)

Also, I think I need to close the [smtp PR](https://github.com/home-assistant/home-assistant.github.io/pull/2514) (was accidentally merged with this one) and make it again. I don't know how to remove files from the PR, sorry!!

Thanks a lot!